src/doc/src/reference/environment-variables.md

```diff
@@ -225,7 +225,8 @@ Note that if one of these values is not provided in the manifest, the
 corresponding environment variable is set to the empty string, `""`.
 
 * `CARGO` --- Path to the `cargo` binary performing the build.
-* `CARGO_MANIFEST_DIR` --- The directory containing the manifest of your package.
+* `CARGO_MANIFEST_DIR` --- The directory containing the manifest of your package. When running a [Cargo script], this variable will be set to the script location.
```

Comment:
That wording change is ambiguous.  To me, it reads as we change the meaning to point to `CARGO_MANIFEST_PATH`.

I don't think this change is needed and it is unrelated to the intent of this PR, so let's remove it.

Reply:
Oops the second sentence should've been in `CARGO_MANIFEST_PATH` let me fix that. Accidentally put it to the DIR instead. ([fix diff](https://github.com/rust-lang/cargo/compare/994083b14c1b9905a242419c15246be0ee5e0d23..d9e1028c1f77b6f11c0227fcd1de967008e87389))
LMK if you still think it's better to just remove it

Reply:
Let's leave it off for now.  We'll need to decide how we document cargo script.  Besides, most unstable documentation gets left in https://doc.rust-lang.org/cargo/reference/unstable.html